Solutions/VMware vCenter/Analytic Rules/vCenterUnusualLogin.yaml

```diff
@@ -0,0 +1,37 @@
+id: 03e8a895-b5ba-49a0-aed3-f9a997d92fbe
+name: vCenter - Unusual login
+description: |
+  'Detects when user login from uncommon IP address.'
+severity: High
+requiredDataConnectors:
+  - connectorId: vCenter
+    dataTypes:
+      - vCenter
+queryFrequency: 1h
+queryPeriod: 14d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - InitialAccess
+  - PrivilegeEscalation
+relevantTechniques:
+  - T1078
+query: |
+  let p_lookback = 14d;
+  let t_lookback = 1h;
+  let root_ips = vCenter
+  | where TimeGenerated between (ago(p_lookback) .. ago(t_lookback))
+  | where EventType has_all ('UserLoginSessionEvent', 'root', 'logged in')
+  | summarize make_set(SourceIP,128);
+  vCenter
+  | where TimeGenerated > ago(t_lookback)
+  | where EventType   has_all ('UserLoginSessionEvent', 'root', 'logged in')
+  | where SourceIP   !in (root_ips)
+  | extend IPCustomEntity = SourceIP
```

Comment:
Do not use custom entity mapping, it is legacy and can cause issues with the rules wizard.